chsdi/tests/functional/test_vector.py

```diff
@@ -30,3 +30,11 @@ def test_attribute_fallback_to_de(self):
         attr = 'toto_de'
 
         self.assertEqual('toto_de', getFallbackLangMatch(queryableAttrs, suffix, attr, suffixAttr))
+
+    def test_test(self):
+        queryableAttrs = ['_frtoto', 'toto_it', 'toto_de']
+        suffix = '_it'
+        suffixAttr = '_de'
```

Comment:
the attribute suffix should be found in the attribute. See https://github.com/geoadmin/mf-chsdi3/blob/master/chsdi/models/vector/__init__.py#L192 on how we generate this suffixAttr. (it is virtually impossible)
What we should do it move the code from line 188 to 200, in `getFallbackLangMatch`. This would make the fuction more self contained.